package.json

```diff
@@ -14,8 +14,8 @@
     "webservice": "./scripts/run-webservice-script.sh",
     "prebuild": "ts-node -O '{\"module\":\"commonjs\"}' git.version.ts && ./scripts/generate-openapi-script.sh",
     "prebuild.prod": "npm run prebuild",
-    "build.prod": "npx ng build --prod --subresource-integrity",
-    "build": "npx ng build --subresource-integrity",
+    "build.prod": "npx ng build -c production --subresource-integrity",
+    "build": "npx ng build -c production --subresource-integrity",
```

Comment:
This is only used for builds running on developers' machines.

Do we want this there? Might be OK, just checking. Presumably it's a slower build?

Reply:
I saw Gary's comment after I submitted my review:

> Angular actually defaulted to prod build on newly generated projects so this is fine (to always prod build).

So it looks like it's fine.